shell/app/atom_content_client.cc

```diff
@@ -200,30 +206,19 @@ base::RefCountedMemory* AtomContentClient::GetDataResourceBytes(
 }
 
 void AtomContentClient::AddAdditionalSchemes(Schemes* schemes) {
-  std::vector<std::string> splited;
```

Comment:
nittiest of nits: `split_args` maybe?

Reply:
That variable was replaced with `tokens` in the PR. Is that an improvement?

Reply:
on second thought that's also fine :)